docs/conceptual/core/codec.md

```diff
@@ -1,13 +1,178 @@
 ---
-title: Parity Scale Codec
+title: SCALE Codec
 ---
 
-* What is parity scale codec?
+The SCALE (Simple Concatenated Aggregate Little-Endian) Codec is a lightweight,
+efficient, binary serialization and deserialization codec.
 
-* How does it work?
+It is designed for high-performance, copy-free encoding and decoding of data in
+resource-constrained execution contexts, like the [Substrate
+runtime](conceptual/runtime/index.md). It is not self-describing in any way and
+assumes the decoding context has all type knowledge about the encoded data.
 
-* Examples
+## SCALE for Substrate
 
-* Why did we invent our own codec?
+Substrate uses the
+[`parity-scale-codec`](https://github.com/paritytech/parity-scale-codec), a Rust
+implementation of the SCALE Codec. This library and the SCALE codec are
+advantageous for Substrate and blockchain systems because:
 
-* What implementations exist?
\ No newline at end of file
+* It does not use Rust STD, and thus can compile to Wasm.
```

Comment:
Serde can be compiled as well without `std`^^

Reply:
I think it is still fair to call out no? I don't assume all codec libraries can compile with `no_std` right?

This one can :)

Reply:
I have heard that our main problem with serde, that you could name here, is a sh*t ton of boilerplate that it produces and bloats the size of the binary apparently?

Reply:
```suggestion
* it is lightweight relative to generic serialization frameworks like [serde](https://serde.rs/), which add significant boilerplate that often bloats the size of the binary
```